src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -103,23 +103,23 @@
   "allDeclaredFields":true,
   "allDeclaredMethods":true,
   "methods":[
-    {"name":"createArrayNode","parameterTypes":[] }, 
```

Comment:
Changes in this file are mostly whitespace-related, except for the final bit: the XML related classes that we manually added last time are no longer necessary - native-image properly takes care of those classes at compile-time, without the need for extra config